supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift-container-storage.adoc

```diff
@@ -0,0 +1,145 @@
+[[openshift-container-storage-conventions]]
+
+[discrete]
+[[backing-store]]
+==== backing store (noun)
+*Description*: A type of storage resource for Multicloud Object Gateway, which is used to store the data like RADOS gateway (RGW), Amazon Web Services S3, Azure Blob Storage, IBM Cloud Object Storage, and so on. 
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[container-storage-interface]]
+==== Container Storage Interface (noun)
+*Description*: The Container Storage Interface (CSI) is a standard for exposing arbitrary block and file storage systems to containerized workloads on Container Orchestration Systems like Kubernetes, and in particular Red Hat OpenShift Container Platform. This allows OpenShift Container Platform to consume storage from third-party storage providers that implement the CSI interface as persistent storage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[multicloud-object-gateway]]
+==== Multicloud Object Gateway (noun)
+*Description*: Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform that allows users to start small and then scale as needed on-premise, in multiple clusters, and with cloud-native storage.
```

Comment:
- "allows" used in this way is kind of anthropomorphic.
- "as needed" probably comes with the same problems as "as expected" for which there is an open issue about #83.
- Also, "start small" is a bit marketing-y and vague here. Does it mean -deploy a small environment-? If so maybe we could say:

`Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform that you can use to start with a small environment and then scale according to your requirements on-premise, in multiple clusters, and with cloud-native storage.`
or
`Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform that you can use to start with a small storage service and then scale according to your requirements on-premise, in multiple clusters, and with cloud-native storage.`

or something else that's more accurate.

Reply:
>     * "as needed" probably comes with the same problems as "as expected" for which there is an open issue about ["As expected" #83](https://github.com/redhat-documentation/supplementary-style-guide/issues/83).

I agree. I would replace "as needed" by "as your scenario requires", for example.

Reply:
Changed.